test-data/unit/check-unreachable-code.test

```diff
@@ -1416,3 +1492,10 @@ x = 1  # E: Statement is unreachable
 raise Exception
 x = 1  # E: Statement is unreachable
 [builtins fixtures/exception.pyi]
+
+[case testUnreachableModuleBody3]
+# flags: --warn-unreachable
+raise Exception
+raise Exception
+x = 1  # E: Statement is unreachable
```

Comment:
Should we instead show the error on the second raise statement?

Reply:
Done! 👍